diagnostics/albedofb/doc/surface_albedo_feedback.rst

```diff
@@ -0,0 +1,57 @@
+Surface Albedo Feedback
+================================
+
+Last update: 1/26/2021
+
+This POD calculates the top of atmosphere radiative impact of surface albedo changes associated with Arctic sea ice loss under global warming. Four quantities are calculated: 
+
+- **Climatological surface albedo**: the ratio of upwelling to downwelling shortwave radiation at the surface expressed as a :math:`\%`.
+
+- **Radiative sensitivity**: the change in reflected shortwave radiation at the top of the atmosphere (in W :math:`m^{-2}`) per  0.01 increase in surface albedo. This quantity, akin to a surface albedo radiative kernel, is calculated from the climatological radiative fluxes at the top of atmosphere and surface using a simplified shortwave radiation model. Higher values generally correspond to less cloudy mean states.
+
+- **Ice sensitivity** : the change in surface albedo normalized by the global mean surface temperature change. This quantity is equal the change in sea ice concentration times the average albedo contrast between ocean and sea ice. The model values are calculated from :math:`4XCO_{2}` simulations and observations are calculated from trends over the historic satellite record.
+
+- **Radiative impact of sea ice loss** : the change in top of atmosphere radiation (positive downward) due to changes in surface albedo normalized by global mean surface temperature change, in in W :math:`m^{-2}` :math:`K^{-1}`. This quantity is equal to the product of the **Radiative sensitivity** and **Ice sensitivity**. The global and annual average of this quantity is equal to the ice albedo feedback of the Arctic.

```

Comment:
Hi @lettie-roach - great descriptions and very clear.  

I appreciate that we want to wrap this POD up, but could we ask you add a few sentences about how a model developer who many not necessarily be an expert in this area should interpret these quantities?   What should they be looking for in each case?  For example, perhaps there are common biases that tend to show up?  Some overall guidance on _how_ to interpret the results would be very useful.

Reply:
I've added more guidance to the rst file

Reply:
Thank you! This looks good.